react/Modal/index.jsx

```diff
@@ -82,5 +83,5 @@ Modal.defaultProps = {
   withCross: true
 }
 
-export { styles }
+export { Content }
```

Comment:
Avoiding the double import is actually a side effect (a nice one). To me it was also to hide the organization of the folders to the end developer.